ui/apps/platform/src/Containers/MainPage/Navigation/HorizontalSubnav.tsx

```diff
@@ -99,15 +99,15 @@ function getSubnavDescriptionGroups(
                               type: 'link',
                               content: 'All vulnerable images',
                               description:
-                                  'View findings for user, platform, and inactive images simultaneously',
```

Comment:
Aha, I had missed a potential ambiguity with **View findings**:
* View (verb) findings (direct object)
* View (noun) findings (oops, back up and read as intended)